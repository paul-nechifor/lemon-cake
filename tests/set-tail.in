(= l :(1 2 3))
(set-tail l ())
l
(set-tail l :(7))
l
(set-tail (tail l) :(8))
l
