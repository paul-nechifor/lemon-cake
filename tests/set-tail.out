(1 2 3)
()
(1)
(7)
(1 7)
(8)
(1 7 8)
